docs/project/doc-style.md

```diff
@@ -93,9 +93,9 @@ More info from the always handy [Grammar Girl site](http://www.quickanddirtytips
 
 ### Pronouns
 
-It's okay to use first and second person pronouns. Specifically, use "we" to
+It's okay to use first and second person pronouns, especially if it lets you avoid a passive construction. Specifically, always use "we" to
 refer to Docker and "you" to refer to the user. For example, "We built the
-`exec` command so you can resize a TTY session."
+`exec` command so you can resize a TTY session." That said, in general, try to write simple, imperative sentences that avoid the use of pronouns altogether. Say "Now, enter your SSH key" rather than "You can now enter your SSH key."
```

Comment:
No, AP says period goes inside quotes (though I think it's more common to have period outside quotes in Euro text, so that's probably why it looks funny to you!).

TBH, I've been slacking off on 80 characters because it doesn't seem like it's being enforced anymore. At some point, we should probably change this guide, but TBH I don't feel like having the discussion at the moment. :-)